src/lib/sticky-header/sticky-header.spec.ts

```diff
@@ -0,0 +1,175 @@
+import {async, ComponentFixture, TestBed, fakeAsync, tick} from '@angular/core/testing';
+import {Component, DebugElement, ViewChild} from '@angular/core';
+import {StickyHeaderModule, CdkStickyRegion, CdkStickyHeader,
+  STICKY_HEADER_SUPPORT_STRATEGY} from './index';
+import {OverlayModule, Scrollable} from '../core/overlay/index';
+import {PlatformModule} from '../core/platform/index';
+import {By} from '@angular/platform-browser';
+import {dispatchFakeEvent} from '@angular/cdk/testing';
+
+
+
+describe('sticky-header with positioning not supported', () => {
+  let fixture: ComponentFixture<StickyHeaderTest>;
+  let testComponent: StickyHeaderTest;
+  let stickyElement: DebugElement;
+  let stickyParentElement: DebugElement;
+  let scrollableElement: DebugElement;
+  let stickyHeaderDir: CdkStickyHeader;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [ OverlayModule, PlatformModule, StickyHeaderModule ],
+      declarations: [StickyHeaderTest],
+      providers: [
+        {provide: STICKY_HEADER_SUPPORT_STRATEGY, useValue: false},
+      ],
+    });
+    TestBed.compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(StickyHeaderTest);
+    fixture.detectChanges();
+    testComponent = fixture.debugElement.componentInstance;
+    stickyElement = fixture.debugElement.query(By.directive(CdkStickyHeader));
+    stickyParentElement = fixture.debugElement.query(By.directive(CdkStickyRegion));
+    stickyHeaderDir = stickyElement.injector.get<CdkStickyHeader>(CdkStickyHeader);
+    scrollableElement = fixture.debugElement.query(By.directive(Scrollable));
+  });
+
+  it('should be able to find stickyParent when sticky positioning is not supported', () => {
```

Comment:
You can remove ` when sticky positioning is not supported` since it's in `describe`